### PR TITLE
GMRQ docs

### PR DESCRIPTION
**Pull request by cxhernandez**
Here's a first crack at this. Let me know what you think, and if I should include Osprey in here.

### CODE REVIEW

File: docs/gmrq.rst [hunk 1]
```diff
@@ -1,8 +1,33 @@
 .. _gmrq:
 
-Model Selection (GMRQ)
+Model Selection using GMRQ
```
Comment (mpharrigan) [inline docs/gmrq.rst]: Check out the sidebar on msmbuilder.org. I like the idea of each page title being of the form
> High level description (technical acronym)

We tell people a lot about "GMRQ"; It should be immediately apparent where they can get a description of what this mysterious 4LA is

Also, if you want to change the title, adjust the underbar to be the same width

File: docs/gmrq.rst [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
 ======================
 
-GMRQ uses a variational principle to help you select model parameters.
+The generalized matrix Rayleigh quotient (GMRQ) is a specific application of
+the variational principle (adapted from `quantum mechanics <https://en.wikipedia.org/wiki/Variational_method_(quantum_mechanics)>_`)
```
Comment (mpharrigan) [inline docs/gmrq.rst]: Is this valid syntax for links in rst?

**Comment by mpharrigan** [comment]
Can we include api reference here?

Maybe we can add the gmrq vs. states plot from @rmcgibbo

File: docs/gmrq.rst (hunk 2 above)
Comment (cxhernandez) [inline docs/gmrq.rst]: It's what's used for the citation link.... Is this not right?

File: docs/gmrq.rst (hunk 1 above)
Comment (cxhernandez) [inline docs/gmrq.rst]: I guess. I just don't think GMRQ is the end-all-be-all for this. Either this section is about GMRQ or the broader topic of model selection, which can be expanded as more methods are developed.

**Comment by cxhernandez** [comment]
note to self: consolidate CT-MSM section with MSM, get rid of `GMRQ` in sidebar, curate the ordering in sidebar.

File: docs/gmrq.rst (hunk 2 above)
Comment (mpharrigan) [inline docs/gmrq.rst]: The github render shows it weird. Maybe theres not supposed to be a space before `<`?

**Comment by mpharrigan** [comment]
fixes #781

**Comment by mpharrigan** [comment]
I'm going to work off of this and try to add an api reference section to this text

**Pull request closed without merging**